.github/workflows/build-assets.yml

```diff
@@ -0,0 +1,210 @@
+name: Build Assets newbuild
+
+env:
+  VCPKG_BINARY_SOURCES: "clear;nuget,GitHub,read"
+  VCPKG_NUGET_REPOSITORY: "https://github.com/hpcc-systems/vcpkg"
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
* Should be removed before master upmerge

Reply:
Looks good

Reply:
There are two additional OS: 1) Ubuntu 22.10. Need to test if the latest fix resolves all the build issues 2) AWS Amazan Linux 2 (amzn2) . This is required for Tim's HMS application in AWS (hpccsystems playground app). It probably is not available in github action.